tests/NHSD.BuyingCatalogue.Solutions.API.UnitTests/Hostings/HybridHostingTypeControllerTests.cs

```diff
@@ -93,5 +107,52 @@ public async Task NullHostingShouldReturnNull()
             result.HostingModel.Should().BeNull();
             result.RequiresHSCN.Should().BeEmpty();
         }
+
+        [Test]
+        public async Task UpdateHybridHostingTypeDetails()
+        {
+            var request = new UpdateHybridHostingTypeViewModel();
+
+            var result =
+                (await _hybridHostingTypeController.Update(_solutionId, request)
+                    .ConfigureAwait(false)) as NoContentResult;
+            result.StatusCode.Should().Be((int)HttpStatusCode.NoContent);
+            _mediatorMock.Verify(
+                x => x.Send(
+                    It.Is<UpdateHybridHostingTypeCommand>(c =>
+                        c.Id == _solutionId &&
+                        c.Data == request
+                    ),
+                    It.IsAny<CancellationToken>()), Times.Once);
```

Comment:
you can also verify that nothing else has been called by using the following:
_mediatorMock.VerifyNoOtherCalls()